Dockerfile

```diff
@@ -23,20 +25,30 @@ ENV GOPROXY=$goproxy
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
+
 # Cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
```

Comment:
Probably worth caching /go/pkg/mod as well, and mounting the go build cache along with this... not sure if `go mod download` would be building any of the mods...